docs/questions/query-builder/expressions-list.md

```diff
@@ -270,7 +310,7 @@ Related: [contains](#contains) and [startswith](#startswith).
 
 ### exp
 
-Returns [Euler's number](https://en.wikipedia.org/wiki/E_(mathematical_constant)), e, raised to the power of the supplied number. (Euler sounds like "Oy-ler").
+Returns [Euler's number](<https://en.wikipedia.org/wiki/E_(mathematical_constant)>), e, raised to the power of the supplied number. (Euler sounds like "Oy-ler").
```

Comment:
I had a math teacher who was extremely passionate about the pronounciation of OY-ler and kil-OM-metre. I have retained nothing else from that class.